### ISSUE TITLE
Add unidirectional relationship field generation to generated schema

### ISSUE THREAD

**Issue by craicoverflow** [aerogear/graphback#614]
Add unidirectional relationship field mappings to generated schema.

```gql
type Note {
  id: ID!
  comments: [Comment]
}

type Comment {
  id: ID!
  text: String
}
```

should be generated as:

```gql
type Note {
  id: ID!
  comments: [Comment]
}

type Comment {
  id: ID!
  text: String
  note: Note! # generated field
}
```

**Comment by craicoverflow** [comment]
Added in https://github.com/aerogear/graphback/pull/632

**Comment by craicoverflow** [comment]
Resolved by #632
